### DISCUSSION TITLE
No Fiscal Year End on YFinance

### DISCUSSION THREAD

**Discussion by Ciccioposticcio** [ranaroussi/yfinance#2116] [Yahoo's data]
Hi all, i think would be useful to add the possibility to get the date of Fiscal Year End for every stock. I found no easy way to get it from the yfinance dataframe. This info is present on the balance sheet on Yahoo Finance, so i think could be an useful information to add on YFinance.

Best regards,
Roberto

**Comment by ValueRaider** [comment]
> This info is present on the balance sheet on Yahoo Finance

Then yfinance has it already
